apps/picsa-tools/manual-tool/src/app/components/material.module.ts

```diff
@@ -17,11 +17,20 @@ export class ManualToolMaterialModule {
   }
 
   registerIcons() {
-    const CUSTOM_ICONS: Record<string, string> = {};
+    const CUSTOM_ICONS: Record<string, string> = {
+      calender: 'calender',
+      campus: 'campus',
+      crop: 'crop',
+      livestock: 'livestock',
+      resource_allocation: 'resourceAllocation',
```

Comment:
nit(non-blocking)
Would be good to keep naming consistency, all icons should be lowercase using underscores

Reply:
Sure